app/models/sales_log.rb

```diff
@@ -47,4 +47,8 @@ def not_started?
   def completed?
     status == "completed"
   end
+
+  def setup_completed?
+    form.setup_sections.first.subsections.first.status(self) == :completed
```

Comment:
Is it enough to check the first section? Do we not need to check that all sections and subsections are complete? 

```suggestion
    form.setup_sections.all? { |sections| sections.subsections.all? { |subsection| subsection.status(self) == :completed }}
```

Reply:
Hm, you're right. I didn't think about it cause we only have 1 atm 🤦